README.md

```diff
@@ -197,99 +196,53 @@ the interval `[0,1)`, with anything less than `0.2` then set to 0 (in order to s
-sort               ----       171.702          ----          ----
-Linux(x86_64), Python 3.10.4, Numpy 1.22.4, Numba 0.55.2, Pandas 1.4.3
-```
 
-```text
-function         ufunc         numpy         numba         weave
------------------------------------------------------------------
-sum              30.985         1.684         1.116         1.350
-prod             32.553        32.269         0.996         1.172
-amin             34.954        34.837         0.989         2.068
-amax             34.580        34.695         1.023         2.132
-len              30.611         1.342         0.805         1.003
-all              36.724         4.355         1.366         1.407
-any              34.570         7.181         1.373         1.410
-anynan           30.840         2.611         0.986         2.557
-allnan           32.463         6.636         0.982         2.562
-mean               ----         2.248         0.985         1.191
-std                ----         6.532         1.084         1.378
-var                ----         6.590         1.086         1.380
-first              ----         2.126         1.033         1.132
-last               ----         1.592         0.957         1.002
-argmax             ----        34.903         1.018          ----
-argmin             ----        38.538         0.996          ----
-nansum             ----         5.148         1.785         1.335
-nanprod            ----        29.445         1.760         1.334
-nanmin             ----        31.752         1.992         2.747
-nanmax             ----        32.247         2.021         2.802
-nanlen             ----         5.099         1.909         1.267
-nanall             ----         9.637         1.826         1.375
-nanany             ----        10.520         1.830         1.384
-nanmean            ----         5.775         2.018         1.430
-nanvar             ----        10.171         2.145         1.640
-nanstd             ----        10.155         2.163         1.637
-nanfirst           ----         5.640         2.201         1.156
-nanlast            ----         5.218         1.734         1.137
-nanargmin          ----        43.795         1.987          ----
-nanargmax          ----        40.354         2.029          ----
-cumsum             ----       138.660         1.270          ----
-cumprod            ----          ----         1.292          ----
-cummax             ----          ----         1.216          ----
-cummin             ----          ----         1.205          ----
-arbitrary          ----       224.213        80.039          ----
-sort               ----       268.514          ----          ----
-Linux(x86_64), Python 2.7.18, Numpy 1.16.6, Numba 0.46.0, Weave 0.17.0
-```
+| function  | ufunc   | numpy   | numba   | pandas  |
+|-----------|---------|---------|---------|---------|
+| sum       | 1.950   | 1.728   | 0.708   | 11.832  |
```

Comment:
This si amazing! The `ufunc` is down to 1.XX from 3X.XX!

Reply:
Indeed. Mb `all` and `any` could be improved from our side now.